### PR TITLE
Increase number of empty drops

### PR DESCRIPTION
**Pull request by aerlaut**
# Description
Increase number of cells needed for empty drops

Tested in a dataset with number of cells too low for `emptyDrops`: 

image

# Details
#### URL to issue
N/A

#### Link to staging deployment URL (or set N/A)
N/A

#### Links to any PRs or resources related to this PR
N/A

#### Integration test branch
master

# Merge checklist
Your changes will be ready for merging after all of the steps below have been completed.

### Code updates
Have best practices and ongoing refactors being observed in this PR
- [ ] Migrated any selector / reducer used to the new format.

### Manual/unit testing
- [ ] Tested changes using InfraMock locally **or** no tests required for change, e.g. Kubernetes chart updates.
- [ ] Validated that current unit tests for code work as expected and are sufficient for code coverage **or** no unit tests required for change, e.g. documentation update.
- [ ] Unit tests written **or** no unit tests required for change, e.g. documentation update.

### Integration testing
**You must check the box below** to run integration tests on the latest commit on your PR branch.
Integration tests have to pass before the PR can be merged. Without checking the box, your PR
**will not pass** the required status checks for merging.

- [ ] Started end-to-end tests on the latest commit.

### Documentation updates
- [ ] Relevant Github READMEs updated **or** no GitHub README updates required.
- [ ] Relevant Wiki pages created/updated **or** no Wiki updates required.

### Optional
- [ ] Staging environment is unstaged before merging.
- [ ] Photo of a cute animal attached to this PR.

### CODE REVIEW

File: pipeline-runner/R/gem2s-3-run_emptydrops.R [hunk 1]
```diff
@@ -46,11 +46,15 @@ compute_sample_edrops <- function(sample_counts) {
 
   if (num_empty_drops < gem2s$max.empty.drops) {
     message("Detected sample as filtered --> Skipping emptyDrops.")
-    sample_edrops <- NULL
-  } else {
-    set.seed(RANDOM_SEED)
-    sample_edrops <- DropletUtils::emptyDrops(sample_counts)
+    return(NULL)
```
Comment (saracastel) [inline pipeline-runner/R/gem2s-3-run_emptydrops.R:49]: why did you remove the `else`?

Comment (saracastel) [inline pipeline-runner/R/gem2s-3-run_emptydrops.R:49]: shouldn't it still be `sample_edrops <- NULL`?

Comment (aerlaut) [reply pipeline-runner/R/gem2s-3-run_emptydrops.R:49]: I change the `if ... else... ` structure into a guard clause pattern:

   if (condition) {
     # do something
      return
   }

   # if not condition
   do something
   return

Comment (aerlaut) [reply pipeline-runner/R/gem2s-3-run_emptydrops.R:49]: This is the same as

    sample_edrops <- NULL
    return(sample_edrops)
